components/ide-proxy/static/code/marketplace.json

```diff
@@ -21,6 +21,20 @@
             "disallowInstall": true,
             "additionalInfo": "Please use the built-in [Settings Sync](https://aka.ms/vscode-settings-sync-help) functionality instead."
         },
+        "tiehuis.zig": {
+			"disallowInstall": true,
```

Comment:
needs :lipstick:

Reply:
Oh... 🤦, I will remember to finally format next time, I swear 😆. Thx!